src/AlgoliaEngine.php

```diff
@@ -56,38 +55,33 @@ public function delete($indexName)
         return true;
     }
 
-    public function search($query, $indexName, $page = 1, $nbResults = null, array $parameters = [])
+    public function search($query, $indexName, $page = 1, $nbResults = 20, $requestOptions = [])
```

Comment:
Maybe all the optional parameters can be send over request options? So we don't need this:
```
        $requestOptions = array_merge($requestOptions, [
	            'hitsPerPage' => $nbResults,
	            'page' => $page - 1,
	        ]);
```